src/test/java/org/broadinstitute/consent/http/service/DataAccessRequestServiceTest.java

```diff
@@ -56,6 +55,10 @@
 @ExtendWith(MockitoExtension.class)
 class DataAccessRequestServiceTest {
 
+  private static final String PI_EMAIL = "pi@example.broadinstitute.org";
+  private static final String SO_EMAIL = "so@example.broadinstitute.org";
+  private static final String IT_EMAIL = "it@example.broadinstitute.org";
+  private final List<UserRole> roles = Collections.singletonList(UserRoles.Researcher());
```

Comment:
In new code I think we should be using the newer `List.of()` API
```suggestion
  private final List<UserRole> roles = List.of(UserRoles.Researcher());
```